cvat/settings/testing_rest.py

```diff
@@ -11,4 +11,6 @@
     "django.contrib.auth.hashers.MD5PasswordHasher",
 ]
 
-IMPORT_CACHE_CLEAN_DELAY = timedelta(seconds=30)
+IMPORT_CACHE_CLEAN_DELAY = timedelta(seconds=1)
```

Comment:
Why did you change it to 1 second? It means that the cleaning job will start after 1 second after the file upload was finished, not imported. I guess that the tests are passed now because the interval is more than IMPORT_CACHE_CLEAN_DELAY. Logically it looks wrong to me.

Reply:
I think we can set 15s here for two reasons:
1. we don't currently have huge test data to import
2. 15 is the standard timeout for test cases

Reply:
Set 15s for IMPORT_CACHE_CLEAN_DELAY and 10s for scheduler interval.